docker/docker.go

```diff
@@ -111,7 +111,7 @@ func main() {
 			*flTls = true
 			cert, err := tls.LoadX509KeyPair(*flCert, *flKey)
 			if err != nil {
-				log.Fatalf("Couldn't load X509 key pair: %s. Key encrypted?", err)
+				log.Fatalf("Couldn't load X509 key pair: %s. Key encrypted", err)
```

Comment:
Based on @duglin s comment, perhaps change to `Is the key encrypted?` or `%s, key may be encrypted`. Apparently, the current message is confusing (even with the question mark)

Should `%s` be changed to `%q` to make it stand out more?